README.md

```diff
@@ -10,40 +10,30 @@ Both are created, run and managed by [ReDI School of Digital Integration](https:
 
 ## Getting started for developers
 
-1. Make sure [MongoDB](https://docs.mongodb.com/manual/installation/) is installed on your computer.
-2. Make sure you're running the _v14.17.6_ version of Node locally when you're running the backend server and checkout to a new branch. To do this, you can use [nvm](https://github.com/nvm-sh/nvm#node-version-manager), which allows you to select different versions of Node via the command line. Alternatively, we have added support for [Volta](https://docs.volta.sh/guide/understanding). So if you choose, you can use Volta, which sets project-defined tools automatically.
-3. Request information for environment variables from @ericbolikowski.
-4. Run `yarn` in project root to install dependencies.
-5. If you're re-seeding, drop the database files: `rm -rf mongodb-data/*` and run `yarn seed` to seed the database.
-
-   Alternatively, you can drop the database in the MongoDB Shell - [mongo](https://www.mongodb.com/docs/v4.4/mongo/) or [mongosh](https://www.mongodb.com/docs/mongodb-shell/). In a new terminal window run the following:
-
-   ```
-   $ mongo
-   $ use mongodb
-   $ db.dropDatabase()
-   ```
+First of all, ReDI Connect / Talent Pool connect to a Salesforce instance via the nestjs-api app. You'll need to set a number of environment variables in your `.env` file to make the Salesforce connection work. Reach out to @helloanil, @katamatata or @ericbolikowski to get set up.
 
-   ~~Then run `yarn seed` in project root to seed the database.~~
+After you've set up `.env`, make sure to update `NX_DEV_MODE_EMAIL_RECIPIENT` to your own email address to receive emails from the platform.
 
-6. Run `yarn start:all` to boot all apps, _or_ a subset of apps using the `start:x` commands in package.json.
+1. Make sure [MongoDB](https://docs.mongodb.com/manual/installation/) is installed on your computer.
+2. Make sure you're running the _v14.17.6_ version of Node locally when you're running the backend server and checkout to a new branch. To do this, you can use [nvm](https://github.com/nvm-sh/nvm#node-version-manager), which allows you to select different versions of Node via the command line. Alternatively, we have added support for [Volta](https://docs.volta.sh/guide/understanding). So if you choose, you can use Volta, which sets project-defined tools automatically.
+3. Run `yarn` in project root to install dependencies.
+4. Run `yarn start:all` to boot all apps, _or_ a subset of apps using the `start:x` commands in package.json.
 
    It'll take a while and lots of warnings will show until everything's booted.
 
-7. If you don't have environment variables set up, talk to @ericbolikowski to get these. The NestJS backend will not work without them.
-
-8. See the [Onboarding Checklist](https://github.com/talent-connect/connect/wiki#onboarding-checklist) in our Wiki.
+5. See the [Onboarding Checklist](https://github.com/talent-connect/connect/wiki#onboarding-checklist) in our Wiki.
```

Comment:
We would need to update the Onboarding Checklist and our Wiki page once we have a new Salesforce demo environment set up.